### PR TITLE
DOC: Fix broken pandas link in release notes

### PR DESCRIPTION
**Pull request by gfyoung**
Title is self-explanatory.

### CODE REVIEW

File: doc/release/1.11.0-notes.rst [hunk 1]
```diff
@@ -99,15 +100,13 @@ As a corollary to this change, we no longer prohibit casting between datetimes
...
 the rule for casting from dates to times is no longer ambiguous.
 
-pandas_: http://pandas.pydata.org
-
 ``linalg.norm`` return type changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 The return type of the ``linalg.norm`` function is now floating point without
 exception.  Some of the norm types previously returned integers.
 
 and returns floating results.polynomial fit changes
```
Comment (charris) [inline doc/release/1.11.0-notes.rst]: `and returns floating results.` should be deleted.

**Comment by charris** [comment]
BTW, if you put `[ci skip]` in the commit message the tests will be skipped.

File: doc/release/1.11.0-notes.rst (hunk 1 above)
Comment (gfyoung) [inline doc/release/1.11.0-notes.rst]: Done.

**Comment by gfyoung** [comment]
@charris : Thanks for the tip!

**Comment by charris** [comment]
Thanks @gfyoung .

**Pull request merged**